port/protobuf.h

```diff
@@ -105,6 +97,13 @@ constexpr bool TextParserCanAllowUnknownField() {
   return GOOGLE_PROTOBUF_VERSION >= 3008000;
 }
 
+inline void PrepareTextParser(google::protobuf::TextFormat::Parser& parser) {
+  if (TextParserCanSetRecursionLimit())
```

Comment:
off cause it will not compile, we can inline PrepareTextParser into main code

Reply:
@vitalybuka I dropped commit now, I guess it was wishful thinking.